Code/GraphMol/FileParsers/FileParsers.h

```diff
@@ -286,6 +286,17 @@ RDKIT_FILEPARSERS_EXPORT RWMol *Mol2BlockToMol(
     const std::string &molBlock, bool sanitize = true, bool removeHs = true,
     Mol2Type variant = CORINA, bool cleanupSubstructures = true);
 
+
+RDKIT_FILEPARSERS_EXPORT void ParseExtraLine(std::string extraLine);
+RDKIT_FILEPARSERS_EXPORT Atom *ParseXYZFileAtomLine(std::string atomLine, RDGeom::Point3D &pos, unsigned int line);
```

Comment:
Strings should generally be passed by const reference in order to avoid copying:
```suggestion
RDKIT_FILEPARSERS_EXPORT void ParseExtraLine(const std::string &extraLine);
RDKIT_FILEPARSERS_EXPORT Atom *ParseXYZFileAtomLine(const std::string &atomLine, RDGeom::Point3D &pos, unsigned int line);
```
you will need to update the definitions too.

Reply:
Also: unless the `ParseExtraLine()` function is being used in code outside of XYZParser.cpp, which doesn't seem to be the case, the function should not be in the public API. So you should remove that declaration from this file.